src/types/shortcuts.ts

```diff
@@ -40,7 +40,7 @@ type TriggerInputShape<Category> = Category extends 'deposit' | 'withdraw'
   : Category extends 'swap-deposit'
   ? ZodRawShape & {
       swapFromToken: typeof tokenAmountWithMetadata
-      enableSwapFee: typeof ZodEnableSwapFee
+      enableAppFee: typeof ZodEnableAppFee
```

Comment:
this is backwards incompatible. Are we making a patch? Or delaying a release? We can also support both to keep it BC

Reply:
Planning on making a patch for 1.94.2.